docs/ref/data-hub/types.md

```diff
@@ -1,6 +1,15 @@
-# Types and constants
+# Classes and Constants
```

Comment:
Lowercase "Constants". The sections headings are generally sentence case.

Reply:
fixed